float64 pos_value
float64 vit_value
